### DISCUSSION TITLE
Add support for python 3.12 and numpy 2.0

### DISCUSSION THREAD

**Discussion by JBreidaks** [metno/gridpp#100] [Gridpp settings]
gridpp please add support for python 3.12 and numpy 2.0

**Comment by tnipen** [comment]
Thanks for the heads up. I agree this is important. I will look at it after my holidays (mid August).

**Comment by tnipen** [comment]
I've created gridpp==0.7.1, which sets numpy<2 in its dependencies. I've also built wheels for python3.12.

I will work on allowing numpy2 at some later time.
